deploy/contents/install/app/docker-utils.sh

```diff
@@ -112,6 +121,13 @@ function docker_push_manifest {
     local push_result=0
     while IFS=, read -r docker_name docker_pretty_name
     do
+
+        if  is_docker_tools_subdir_exist "$manifest_dir/$docker_name"; then
```

Comment:
add a comment here why we use such logic here, what does it mean if `"$manifest_dir/$docker_name"` exists and what if `"$manifest_dir/$docker_pretty_name"` exists